metadata-ingestion/tests/test_helpers/state_helpers.py

```diff
@@ -91,3 +99,15 @@ def monkey_patch_get_latest_timeseries_value(
 
     mock_datahub_graph_ctx = MockDataHubGraphContext()
     return mock_datahub_graph_ctx.mock_graph
+
+
+def get_current_checkpoint_from_pipeline(
```

Comment:
Ahh I will def get conflicts here

Reply:
you should be able to accept all of my changes, and then just tweak this `get_current_checkpoint_from_pipeline` method